tests/ipfwd/conftest.py

```diff
@@ -45,15 +46,31 @@ def get_lag_facts(dut, lag_facts, switch_arptable, mg_facts, ignore_lags, enum_f
 
 
 def get_port_facts(dut, mg_facts, port_status, switch_arptable, ignore_intfs, key='src'):
-    if not mg_facts['minigraph_interfaces']:
-        pytest.fail("minigraph_interfaces is not defined.")
+    interfaces = None
+    is_backend_topology = constants.IS_BACKEND_TOPOLOGY_KEY in mg_facts.keys() and mg_facts[constants.IS_BACKEND_TOPOLOGY_KEY]
+    if is_backend_topology:
+        interfaces = mg_facts['minigraph_vlan_sub_interfaces']
+    else:
+        interfaces = mg_facts['minigraph_interfaces']
+
+    if not interfaces:
+        pytest.fail("interfaces is not defined.")
+
     selected_port_facts = {}
     up_port = None
     for a_intf_name, a_intf_data in port_status['int_status'].items():
         if a_intf_data['oper_state'] == 'up' and a_intf_name not in ignore_intfs:
             # Got a port that is up and not already used.
-            for intf in mg_facts['minigraph_interfaces']:
-                if intf['attachto'] == a_intf_name:
+            for intf in interfaces:
+
```

Comment:
minor issue. please remove this empty line

Reply:
Done, removed the empty line